okio/src/test/java/okio/BufferedSinkTest.java

```diff
@@ -211,4 +211,46 @@
     } catch (ArrayIndexOutOfBoundsException expected) {
     }
   }
+
+  @Test public void longDecimalString() throws IOException {
+    assertLongDecimalString(Long.MIN_VALUE);
+    assertLongDecimalString(Long.MIN_VALUE + 1);
+    assertLongDecimalString(Integer.MIN_VALUE - 1L);
+    assertLongDecimalString(Integer.MIN_VALUE);
+    assertLongDecimalString(-1);
+    assertLongDecimalString(0);
+    assertLongDecimalString(1);
+    assertLongDecimalString(Integer.MAX_VALUE);
+    assertLongDecimalString(Integer.MAX_VALUE + 1L);
+    assertLongDecimalString(Long.MAX_VALUE);
+    assertLongDecimalString(Long.MAX_VALUE - 1);
```

Comment:
nit: swap this & previous line? Weird for these to be in _almost_ increasing order.